css/properties.json

```diff
@@ -5833,6 +5833,21 @@
     "order": "lengthOrPercentageBeforeKeywords",
     "status": "standard"
   },
+  "text-justify": {
+    "syntax": "auto | distribute | inter-character | inter-word | none",
+    "media": "visual",
+    "inherited": true,
+    "animationType": "discrete",
+    "percentages": "no",
+    "groups": [
+      "CSS Text"
+    ],
+    "initial": "auto",
+    "appliesto": "allElements",
+    "computed": "asSpecified",
+    "order": "uniqueOrder",
+    "status": "standard"
```

Comment:
We always follow the newest spec, afaik. This is what I tried to do when filling this in.